dim_checks/moz-fx-data-shared-prod/telemetry/rolling_cohorts/dim_checks.yaml

```diff
@@ -0,0 +1,85 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - cohort_date
+          - client_id
+          - activity_segment
+          - app_version
+          - device_model
+          - locale
+          - normalized_app_name
+          - normalized_channel
+          - normalized_os
+          - normalized_os_version
+          - os_version_major
+          - os_version_minor
+    - type: column_length
+      params:
+        columns:
+          - country
+        condition: "= 2"
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_channel
+        expected_values:
+          - nightly
+          - aurora
+          - release
+          - Other
+          - beta
+          - esr
+    - type: value_in_set
+      params:
+        columns:
+          - activity_segment
+        expected_values:
+          - infrequent_user
+          - other
+          - casual_user
+          - regular_user
+          - core_user
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_app_name
+        expected_values:
+          - "Firefox iOS"
+          - Fenix
+          - "Focus Android"
+          - "Focus iOS"
+          - "Firefox Desktop"
+          - "Fenix BrowserStack"
+          - "Firefox iOS BrowserStack"
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_os
+        expected_values:
+          - iOS
+          - Android
+          - Linux
+          - Linux_CXL
+          - Darwin
+          - Windows_95
+          - Windows_98
+          - Windows_NT
+          - OpenBSD
+          - SunOS
+          - HubertOS
+
+    # TODO: Regex check to make sure locale follows: [2 characters]-[2 characters] rule
```

Comment:
Is [this the validation](https://github.com/mozilla/dim/blob/54bc34e71e5af3824429639767adcde9f87f5745/dim_checks/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/dim_checks.yaml#L84) you mean to add here?